### PR TITLE
Add @olo/data-generator package

### PR DESCRIPTION
**Pull request by jacobroufa**
Per conversations had over the last couple months with Lindsay and Greg, this tool built for use with the Design System packages is being open sourced.

### CODE REVIEW

File: packages/olo-gulp-helpers/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "ts-loader": "6.2.1",
     "tslint": "5.9.1",
     "tslint-teamcity-reporter": "2.0.0",
-    "typescript": "2.7",
+    "typescript": "3.9.7",
```
Comment (dustinsoftware) [inline packages/olo-gulp-helpers/package.json:37]: (this is a public repo)

This looks fine, but we'll need to check our internal projects to make sure this package bump won't break any builds.

File: packages/data-generator/package.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@olo/data-generator",
```
Comment (dustinsoftware) [inline packages/data-generator/package.json]: This will fail the publish step since we don't use `@olo` on public npm. Maybe `olo-data-generator`?

**Review by dustinsoftware — COMMENTED** [review]
Looks like a pretty straight forward port..

File: packages/olo-gulp-helpers/package.json (hunk 1 above)
Comment (jacobroufa) [reply packages/olo-gulp-helpers/package.json:37]: Understand. This needed to be bumped to a 3.x version to get the build to pass for `data-generator`, and as such I chose the latest version. Is there some sort of reverse-lookup to see which projects of ours use the `olo-gulp-helpers`?

File: packages/data-generator/package.json (hunk 2 above)
Comment (jacobroufa) [reply packages/data-generator/package.json]: Addressed in rebased 5c23ce5

**Review by dustinsoftware — APPROVED** [review]

**Pull request merged**